### PR TITLE
Feature/todo update on completion

### PR DESCRIPTION
**Pull request by zli213**
#74 #75

### CODE REVIEW

**Comment by ryanwooa** [comment]
Function is OK. 
Here is a little CSS problem.
image

**Comment by ryanwooa** [comment]
image
Completed Task in project have the problem that I told you this afternoon.

File: components/application/widgets/CheckBoxButton.js [hunk 1]
```diff
@@ -10,11 +10,14 @@
  */
 
 "use client";
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, use } from "react";
```
Comment (ryanwooa) [inline components/application/widgets/CheckBoxButton.js:13]: "use" ?

Comment (zli213) [reply components/application/widgets/CheckBoxButton.js]: Done.

**Comment by zli213** [comment]
> image Completed Task in project have the problem that I told you this afternoon.

I haven't made any changes here, and I didn't notice the same issue during my presentation part of the weekend meeting. If you can reproduce the steps, it would help to improve, thank you.

**Comment by ryanwooa** [comment]
> > image Completed Task in project have the problem that I told you this afternoon.
> 
> I haven't made any changes here, and I didn't notice the same issue during my presentation part of the weekend meeting. If you can reproduce the steps, it would help to improve, thank you.

image

**Review by jlmjkfd — APPROVED** [review]

**Review by kesesek — COMMENTED** [review]
An error occurs when the undo button is clicked.

**Comment by ryanwooa** [comment]
image
 Still have some issue

1. the CSS of completed button
2. In the project page, when u click the button to complete a task, the finished task will display in multiple section, like what I circled in the picture, the "asdfa" task.

**Pull request merged**